aadwarf64/aadwarf64.rst

```diff
@@ -362,7 +362,8 @@ integers.
    +----------------+--------------------------+----------------------------------------------------+
    | 31             | SP                       | 64-bit stack pointer                               |
    +----------------+--------------------------+----------------------------------------------------+
-   | 32             | Reserved                 | \-                                                 |
+   | 32             | PC                       | 64-bit program counter                             |
```

Comment:
how is this handled by existing unwinders? will they crash or ignore the related dwarf operations.

Reply:
LLVM/libunwind and savannah/libunwind already accepts it as any other register. gcc/libgcc doesn't and I got pushback on the change without spec change.

Reply:
As there have been no obections to the change, and the one question asked has been answered I am going to accept this pull request.